sdks/python/src/stepflow_py/server.py

```diff
@@ -376,18 +385,166 @@ async def _handle_component_execute(
             traceback.print_exc(file=sys.stderr)
             raise StepflowExecutionError(f"Component execution failed: {str(e)}") from e
 
-    def _add_langchain_support(self):
-        """Add LangChain integration support to the server."""
-        try:
-            from stepflow_py.langchain_components import register_langchain_components
-            from stepflow_py.langchain_registry import add_langchain_component_method
+    def langchain_component(
```

Comment:
Do we need to do any minimal version checking or is that overkill? Save for later too?